src/front/app/components/globals/Container.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+class Container extends Component {
+  static propTypes = {
+    children: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.node,
+      PropTypes.instanceOf(Component),
+    ]).isRequired,
+    className: PropTypes.string,
+    fluid: PropTypes.bool,
+  };
+
+  static defaultProps = {
+    fluid: false,
+  };
+
+  render() {
+    const {
+      children,
+      className,
+      fluid,
+      ...props
+    } = this.props;
+
+    const containerClass = classNames({
+      container: !fluid,
```

Comment:
Doesn't `container` class is required anyway? iirc, when using container fluid you must add them both `container container-fluid`?